### PR TITLE
Replace procrunner with subprocess

### PR DESCRIPTION
**Pull request by dwmoreau**

### CODE REVIEW

**Comment by bkpoon** [comment]
Can the changes to the `txt` files in `libtbx/auto_build/conda_envs` be reverted? I can remove the `procrunner` dependency the next time I update these static environments.

File: libtbx/auto_build/bootstrap.py [hunk 1]
```diff
@@ -1163,10 +1163,6 @@ def __init__(
       hot = self.get_hot()
       if self.use_conda is not None:
         for module in ['scons']:
-          # SCons conda package may cause issues with procrunner on Python 2.7
-          # https://stackoverflow.com/questions/24453387/scons-attributeerror-builtin-function-or-method-object-has-no-attribute-disp
-          if module == 'scons' and self.python == '27':
-            continue
```
Comment (bkpoon) [inline libtbx/auto_build/bootstrap.py:1169]: This change should also be reverted.

Comment (dwmoreau) [reply libtbx/auto_build/bootstrap.py]: This has been reverted in a new commit

**Comment by phyy-nx** [comment]
I have also removed procrunner from xfel_regression

**Review by phyy-nx — APPROVED** [review]

**Pull request merged**